components/style/properties/longhand/inherited_svg.mako.rs

```diff
@@ -101,8 +101,8 @@ ${helpers.predefined_type(
 )}
 
 ${helpers.predefined_type(
-    "stroke-dashoffset", "LengthOrPercentageOrNumber",
-    "Either::First(0.0)",
+    "stroke-dashoffset", "SVGStrokeLength",
+    "computed::SVGStrokeLength(computed::NumberOrPercentage::Number(1.0))",
```

Comment:
This is changing the default value of the stroke length, is it intended? If so, why? I think it deserves a separate commit.